tables/AlternateMans/StokesIStManColumn.h

```diff
@@ -0,0 +1,240 @@
+#ifndef CASACORE_STOKES_I_ST_MAN_COLUMN_H_
+#define CASACORE_STOKES_I_ST_MAN_COLUMN_H_
+
+#include <casacore/tables/DataMan/StManColumn.h>
+
+#include <casacore/casa/Arrays/Array.h>
+#include <casacore/casa/Arrays/IPosition.h>
+
+#include "BufferedColumnarFile.h"
+
+#include <optional>
+
+namespace casacore {
+
+class StokesIStMan;
+
+/**
+ * Expands @p n values from single Stokes I values
+ * to have 4 values, in place. This implies the array
+ * should have place to store n*4 values.
+ */
+template <typename T>
+void ExpandFromStokesI(T *data, size_t n) {
+  for (size_t i = 0; i != n; ++i) {
+    const size_t index = n - i - 1;
+    const T value = data[index];
+    data[index * 4] = value;
+    data[index * 4 + 1] = T();
+    data[index * 4 + 2] = T();
+    data[index * 4 + 3] = value;
+  }
+}
+
+template <typename T>
```

Comment:
Could you insert some documentation here that mentions the factor 0.5?

Reply:
Good point, added. I've now also added a check for non-zero xy/yx values, which should make the dataman a bit safer to use.